src/components/controls/selects/SearchSelect/SearchSelect.stories.tsx

```diff
@@ -184,4 +197,7 @@ Keyboard.play = async ({ canvasElement }) => {
   await userEvent.keyboard("[Enter]")
   await expect(inputField).toHaveValue("")
   await expect(inputField).toHaveFocus()
+  inputField.blur()
```

Comment:
Let's keep this story close to user keyboard behavior and avoid programmatic blur:

```suggestion
  await userEvent.keyboard("[Tab]")
```